src/GitUtils.ps1

```diff
@@ -59,7 +59,7 @@ function Get-GitDirectory {
     }
 }
 
-function Get-GitBranch($gitDir = $(Get-GitDirectory), [Diagnostics.Stopwatch]$sw) {
+function Get-GitBranch($branch = $null, $gitDir = $(Get-GitDirectory), [switch]$isDotGitOrBare, [Diagnostics.Stopwatch]$sw) {
```

Comment:
> This is not an exported command but, in general, I'd add new parameters at the end of the parameter list to avoid breaking any callers that use positional parameters. That said, the only caller is in this file - besides to Pester test calls to this command. So, if it makes more sense for the `Branch` parameter to be first then go for it.

I generally agree, but I wanted to keep `$sw` last so figured all bets are off.

> In retrospect, we probably should have been more consistent in using PascalCasing on all function parameters given that they correspond to command parameters e.g.: `Get-GitBranch -Branch main -GitDir ~\.git`.

Consistency would be good. Can revisit holistically after I get a release out.

Reply:
Yes, it would be better to have style changes and semantic changes in separate PRs.  Makes them both easier to review.  😁